packages/c/crun/pspec_x86_64.xml

```diff
@@ -0,0 +1,56 @@
+<PISI>
+    <Source>
+        <Name>crun</Name>
+        <Homepage>https://github.com/containers/crun</Homepage>
+        <Packager>
+            <Name>Robert Gonzalez</Name>
+            <Email>uni.dos12@outlook.com</Email>
+        </Packager>
+        <License>- GPL-2.0-only
+- LGPL-2.1-only # the library component
+</License>
+        <PartOf>virt</PartOf>
+        <Summary xml:lang="en">A fast and low-memory footprint OCI Container Runtime fully written in C.</Summary>
+        <Description xml:lang="en">- A fast and lightweight fully featured OCI runtime and C library for running containers
+- devel : A C library for running containers
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>crun</Name>
+        <Summary xml:lang="en">A fast and low-memory footprint OCI Container Runtime fully written in C.</Summary>
+        <Description xml:lang="en">- A fast and lightweight fully featured OCI runtime and C library for running containers
+- devel : A C library for running containers
+</Description>
+        <PartOf>virt</PartOf>
+        <Files>
+            <Path fileType="executable">/usr/bin/crun</Path>
+            <Path fileType="library">/usr/lib64/libcrun.so.0</Path>
+            <Path fileType="library">/usr/lib64/libcrun.so.0.0.0</Path>
+            <Path fileType="man">/usr/share/man/man1/crun.1</Path>
+        </Files>
+    </Package>
+    <Package>
+        <Name>crun-devel</Name>
+        <Summary xml:lang="en">Development files for crun</Summary>
+        <Description xml:lang="en">- A fast and lightweight fully featured OCI runtime and C library for running containers
+- devel : A C library for running containers
+</Description>
+        <PartOf>programming.library</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="1">crun</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="library">/usr/lib64/libcrun.so</Path>
```

Comment:
Would it make sense to move these libraries into the main package? The main package already has some libraries.

Reply:
It probably does. I decided to split it up because it provides a utility and a library.